guava/src/com/google/common/primitives/UnsignedBytes.java

```diff
@@ -155,7 +159,7 @@ public static byte min(byte @MinLen(1)... array) {
    *     in the array
    * @throws IllegalArgumentException if {@code array} is empty
    */
-  public static byte max(byte @MinLen(1)... array) {
+  public static @Unsigned byte max(byte @Unsigned @MinLen(1)... array) {
```

Comment:
For min and max (in this file and elsewhere, the`@Unsigned` annotations are on the array, not the byte.  Please move them.

Reply:
Done!

Reply:
You do not need to reply "done".  It clutters the conversation history.  Please do respond to reviews that are confusing/wrong, or that you don't want to do.  If you answer nothing, then that implicitly means that you have addressed the comment.